claim/README.md

```diff
@@ -1,41 +1,223 @@
 <!--
 ---
 title: "Agent claiming"
+description: "Agent claiming is part of the onboarding process when creating a workspace in Netdata Cloud."
```

Comment:
It's not only part of the onboarding process. Users can always claim nodes to their spaces, even after onboarding. Also we use Space instead of Workspace

Reply:
Good point. I've changed this and the first sentence of the doc itself.